packages/examples/src/langium/config/wrapperLangiumClassic.ts

```diff
@@ -15,14 +20,11 @@ export const setupLangiumClientClassic = async (): Promise<UserConfig> => {
         },
         wrapperConfig: {
             serviceConfig: {
-                enableModelService: true,
-                configureEditorOrViewsService: {
+                userServices: {
+                    ...getConfigurationServiceOverride(Uri.file('/workspace')),
+                    ...getEditorServiceOverride(useOpenEditorStub),
+                    ...getKeybindingsServiceOverride()
```

Comment:
Just a note, should document this breaking change.

Reply:
Good point, I documented this in `monaco-languageclient` and just updated the CHANGELOG.

Reply:
Done ⬇️